src/_includes/partials/footer.njk

```diff
@@ -268,6 +268,9 @@
             <a href="https://www.freecodecamp.org/news/copyright-policy/" rel="noopener noreferrer" target="_blank" data-test-label="copyright">
                 {% t 'footer.links.copyright' %}
             </a>
+            <a href="https://hashnode.com/" rel="noopener noreferrer" target="_blank" data-test-label="powered-by">
```

Comment:
We can use a simple `if` statement here so this link is only shown for English builds, and add the "Powered by Hashnode" text in directly for now:

```suggestion
            {% if site.lang === 'en' %}
                <a href="https://hashnode.com/" rel="noopener noreferrer" target="_blank" data-test-label="powered-by">
                    Powered by Hashnode
                </a>
            {% endif %}
```